doc/help/kdb-record.md

```diff
@@ -0,0 +1,144 @@
+
+## COMMON PARAMETERS
+
+Most commands allow specification of the name of the session they should work against.
+If no session name is provided, the active session will be used.
+
+A notable exception is the `start` command.
+If no session name is provided, it will create a random name.
+
+## COMMANDS
+
+### `start [session_name]`
+The `start` command is used to start a new or resume an existing session.
+If the environment variable `ELEKTRA_RECORD_ACTIVE_SESSION` is set, this command has no immediate effect.
+As soon as the environment variable is unset, the session started or resumed with this command will be active.
+
+This command will accept a single optional argument that is the name of the session.
+If the name is not provided, a random name will be generated.
+
+On success, the name of the session will be printed to `stdout`, and the return code is `0`.
+
+### `stop`
+The `stop` command is used to stop the active recording session.
+The session can then be resumed using the `start` command with the appropriate session name.
+Stopping a recording session will not delete any recorded changes.
+
+### `reset [session_name]`
+The `reset` command will clear the specified recording.
+If no session name argument is provided, the currently active session will be reset.
+
+### `delete [session_name]`
+Deletes the specified recording session.
+It the optional argument is not provided, the current session will be deleted.
+If it is the currently active session, it will be stopped.
+
+### `mark [--meta meta_name] <key_name>`
+This command marks a key to be used for validation when importing the recording.
+By default, the values of a marked key will be compared when importing.
+The import will fail if the key has a different value.
+
+Using the `--meta` flag, one can specify to use the value of a metakey instead of the value of the key.
+
+### `list`
+Lists all known recording sessions.
+
+### `name`
+Prints out the name of the currently active recording session.
+If no session is active, the return code is `1`, otherwise `0`.
+
+### `changes [session_name]`
+Shows all the changes in the specified session in human-readable format.
+Outputs to `stdout`.
+If no session is specified, the currently active session is used.
+
+### `export  [--format=replay|diff] [session_name]`
+Exports the changes made in the specified session.
+Outputs to `stdout`.
+If no session is specified, the currently active session is used.
+
+The `export` command has the ability to output two different types: diffs and replays.
+
+The `--format=diff` argument (default) only exports the changes between the start and end states of the recording session.
+This means, e.g., that when you add a key and remove it in the same session, it will not be exported.
+
+The `--format=replay` argument creates a file that contains every modifying operation made in the session in the order they were performed.
```

Comment:
These functionalities are so different that they should be different commands. Maybe the "replay" feature should not even be within the whole "kdb record" framework. (It seems to me that the current description of this man page, with the exception of this `export --format=replay`, can be implemented as a base+ours+assert keyset. Only the replay needs an actual log of everything that happened?)

Reply:
> Only the replay needs an actual log of everything that happened?

See above: https://github.com/ElektraInitiative/libelektra/pull/4399#discussion_r913869936